add required CIDR to the main Prereq section for reducing back-and-forth (#2569)
astronomer/docs: astro/install-aws-hybrid.md

sidebar_custom_props: { icon: 'img/aws.png' }
toc_min_heading_level: 2
toc_max_heading_level: 2

> description: "If you have an existing Amazon Web Services (AWS) instance, use these instructions to complete an Astro installation. This is where you’ll find the prerequisites and the process you’ll need to follow to allow Astronomer support to provision your network resources."

Reviewer: ```suggestion
description: "If you have an existing Amazon Web Services (AWS) instance, use these instructions to complete the installation for Astro Hybrid."
```

For a list of the AWS resources and configurations that Astronomer supports, see [AWS resource reference](resource-reference-aws-hybrid.md). For more information about the shared responsibility model, see [Shared responsibility model](shared-responsibility-model.md).

To complete the installation, you'll:

- Create an Astronomer account.
- Create a new AWS account with the required AWS resources.

> - Create the IAM policies used by Astro. This includes a cross-account IAM role that Astro can assume and [Permissions boundaries for IAM entities](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html).

Reviewer: ```suggestion
- Create the IAM policies used by Astro. This includes a cross-account IAM role that Astro can assume and [permissions boundaries for IAM entities](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html).
```

    The following table lists the required [EC2 service quotas](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-resource-limits.html).

    | QuotaCode  | QuotaName                                                        | Minimum Value  |
    | -----------| ---------------------------------------------------------------- | ---------------|
    | L-1216C47A | Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances | 40             |
    | L-34B43A08 | All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Requests  | 40             |

  These quotas are required to mitigate near term capacity risks and simplify the Astro onboarding experience. Refer to [AWS documentation](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html) to modify or increase a specific quota.

- A CIDR block with a range of `/20`. If you don't have a preferred CIDR block, Astro will provision a VPC using a default of `172.20.0.0/20`. Astro uses this VPC for 2 public subnets and 2 private subnets. See [AWS resource reference](resource-reference-aws-hybrid.md).

> - Admin access to create a stack using [CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-cli-creating-stack.html).

Reviewer: ```suggestion
- Permissions to create a stack using [CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-cli-creating-stack.html).
```
